client/components/backup-storage-space/usage-warning/use-storage-status-text.ts

```diff
@@ -2,6 +2,8 @@ import { useTranslate, TranslateResult } from 'i18n-calypso';
 import { useMemo } from 'react';
 import { StorageUsageLevelName, StorageUsageLevels } from 'calypso/state/rewind/storage/types';
 
+// @TODO: migrate translations to use `@wordpress/i18n` to comply with using core components
+// @see https://github.com/Automattic/wp-calypso/blob/trunk/client/dashboard/docs/i18n.md
```

Comment:
I'm planning to migrate this to use the core i18n components instead, but I need to do it in a separate PR with the String Freeze label so we don't affect the users while waiting for the new translations :)

We may need to wait for new translations probably just on the `StorageUsageLevels.Full` one, since we can use `_n()` for it for pluralization and it will create a new string.